### PR TITLE
Replace OAuth flow for Stripe onboarding with Account Links

### PR DESCRIPTION
**Pull request by mpark-stripe**
Replacing the OAuth flow for Stripe onboarding with Account Links, which is the preferred method to authenticate with Stripe.

### CODE REVIEW

File: server/views/dashboard.pug [hunk 1]
```diff
@@ -23,7 +23,7 @@ block content
       .block.available
         p Your balance
         h1 #{balanceCurrency}#{((balanceAvailable + balancePending) / 100).toFixed(2)}
-        h2  #{balanceCurrency}$#{(balanceAvailable / 100).toFixed(2)} available
+        h2  #{balanceCurrency}#{(balanceAvailable / 100).toFixed(2)} available
```
Comment (nfriend-stripe) [inline server/views/dashboard.pug:26]: Nice catch!

File: server/routes/pilots/stripe.js [hunk 2]
```diff
@@ -20,81 +20,93 @@ function pilotRequired(req, res, next) {
...
-    await req.user.save();
-
-    // Redirect to the Rocket Rides dashboard
-    req.flash('showBanner', 'true');
-    res.redirect('/pilots/dashboard');
   } catch (err) {
-    console.log('The Stripe onboarding process has not succeeded.');
-    next(err);
+    console.log(err);
+    res.redirect('/pilots/signup');
```
Comment (nfriend-stripe) [inline server/routes/pilots/stripe.js:107]: It looks an error in the previous version of this endpoint would result in an error page like this:

image

Which is kind of nice for demo purposes. The new version silently swallows the error and redirects back to the signup page. I think I slightly prefer the old behavior (although I don't feel too strongly about it). WDYT?

File: README.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ To integrate Stripe Connect in your own app, check out these two files in partic
 
 You'll need a Stripe account to manage pilot onboarding and payments:
 - [Sign up for free](https://dashboard.stripe.com/register), then [enable Connect](https://dashboard.stripe.com/account/applications/settings) by filling in your Connect settings.
-- In the **Integration** section, add the following **Redirect URI**: `http://localhost:3000/pilots/stripe/token`.
+- Fill in the necessary information in the **Branding** section in [Connect settings](https://dashboard.stripe.com/test/settings/connect).
```
Comment (nfriend-stripe) [inline README.md:27]: Thanks for adding this 👍

**Review by nfriend-stripe — CHANGES_REQUESTED** [review]
@mpark-stripe This is looking great! 💯  One minor suggestion for you about error handling (which I don't feel strongly about).

One other comment: can we remove the `clientId`, `authorizeUri`, and `tokenUri` keys from `config.js.default` since they're no longer being used?

File: server/routes/pilots/stripe.js (hunk 2 above)
Comment (mpark-stripe) [reply server/routes/pilots/stripe.js:107]: Sure, I think that makes sense to me. Surfacing the error is probably more clear to the user than silently redirecting.

**Review by bramirez-stripe — APPROVED** [review]

**Pull request merged**